### ISSUE TITLE
checkra1n for Linux breaks Touch ID

### ISSUE THREAD

**Issue by xorpix** [checkra1n/BugTracker#777]
**Tell us about your setup:**
1. What iDevice are you using?
iPhone 8 Plus
2. On what version of iOS is it?
iOS 13.3.1
3. What version of checkra1n are you using? GUI or CLI?
CLI
4. What is your host system (OS version? Hackintosh? VM? etc.)?
Manjaro Linux
5. How are you connecting to the device (USB-A? USB-C? Apple/3rd party cable? Through a USB hub?)?
USB-A

**What are the steps to reproduce the issue?**
Just jailbreak with checkra1n 0.9.8 for Linux.

**Does the issue also occur if you enable Safe Mode?**
Yes, it occurs in Safe Mode

Also, i'm following all onscreen directions while jailbreaking. Made already 5 attempts. And each of them led to Touch ID not working.

**Comment by xorpix** [comment]
I was using mac version previously and wanted to test linux version. But seems the issue is gone when i restored RootFS and reinstalled jailbreak.

**Comment by Siguza** [comment]
I'm gonna mark this as invalid since it doesn't seem to be a checkra1n issue.  
But just out of curiosity, did you have Cercube installed? Because there have been reports of that breaking TouchID.

**Comment by xorpix** [comment]
Yes, I did have Cercube. But after restoring rootFS, reinstalling jailbreak using 0.9.8 and recovering all my tweaks (Cercube included) the device came back to normal. Before I’ve done it I experienced Touch ID problem. Also my device was heating up so much and that was also causing a massive battery drain. My thoughts was that 0.9.8 is actually a really major update and by enabling the jailbreak, which was installed using earlier versions, it could somehow mess up and cause those issues. But idk. That’s all just my guessworks.
